pages/home/home.ejs

```diff
@@ -100,6 +100,55 @@
           </tbody>
         </table>
 
+        <table class="card mb-4">
+          <div class="card-header bg-warning text-white">Upcoming Events</div>
+          <div class="card-body">
+            <p>
+              <strong><center>NEXT DUE:</center></strong>
```

Comment:
This can probably be omitted in favor of the card header (the "Upcoming Events" text above)